web/themes/custom/itkdev/itkdev_project_theme/templates/components/new-project-button.html.twig

```diff
@@ -1,4 +1,6 @@
-<a href="/node/add/project" class="font-thin bg-green-700 hover:bg-green-600 text-white py-2 px-4 rounded inline-flex items-center" data-drupal-link-system-path="node/add/project">
+{% set path = path('node.add', {node_type: 'project'}) %}
+
+<a href={{ path }} class="font-thin bg-green-700 hover:bg-green-600 text-white py-2 px-4 rounded inline-flex items-center" data-drupal-link-system-path="node/add/project">
```

Comment:
What is the `data-drupal-link-system-path` attribute used for? Should it be set using `path`?

```suggestion
<a href={{ path }} class="font-thin bg-green-700 hover:bg-green-600 text-white py-2 px-4 rounded inline-flex items-center" data-drupal-link-system-path="{{ path }}">
```